src/content/development/building-testing/ci-maintenance/_index.en.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "CI/CD Maintenance"
+date: 2022-01-05T16:03:26+01:00
+weight: 10
+---
+
+This page documents the maintaince of Submariner's CI/CD for developers.
+
+## Kubernetes Versions
+
+The versions of Kubernetes tested in Submariner's CI need to be updated for new K8s releases.
+
+Submariner's policy is to support all versions upstream-K8s supports and no EOL versions.
```

Comment:
Any reason not to expand "K8s" to “Kubernetes” throughout this doc? https://submariner.io/development/website/style-guide/

Reply:
I had a dev audience in mind, but yes you're right better to be consistent.